src/registrar/assets/js/uswds-edited.js

```diff
@@ -0,0 +1,7035 @@
+/*
+* Ctrl-F DOTGOV for modifications to USWDS compiled code. 
+*/
```

Comment:
**suggestion** please add a comment for why this copy over was needed. No more than a couple sentences, just enough so another dev would understand why this file exists here

Reply:
**question/ nonblocking** Are there really just two changes in this file?

Reply:
Yep, just making the classes available in the window context